pages/content/amp-dev/documentation/guides-and-tutorials/learn/validation-workflow/validate_emails.md

```diff
@@ -50,6 +50,22 @@ amphtml-validator --html_format AMP4EMAIL <amphtml file>
 If the email is valid the command-line tool will result in a `PASS`. If invalid, it will return with the errors it found. 
 
 
+## AMP playground
+
+You can also validate AMP Emails using the [AMP playground](https://playground.amp.dev/?runtime=amp4email). Similar to the web-based validator, paste your AMP Email into the tool, and the playground will flag any validator errors directly inline.
+
+### Validate delivered emails
+
+Sometimes your delivered AMP Emails may be invalid even though the email markup you authored has already been validated by tools documented in this page. The most common reason for this to happen is that your ESP modified your email markup and made it invalid after you have sent your email to your ESP for delivery. For example, if you haven't configured HTTPS tracking pixels with your ESP, then your ESP will add a insecure HTTP tracking pixel to your email. Since AMP Emails only allow HTTPS images, this will make your AMP Email invalid.
```

Comment:
Might be worth adding a sentence in the beginning explaining "ESP" with an example.

Reply:
Added a link to the supported ESP page on amp.dev and also used an example ESP in "For example" sentence. LMK WYT.